prime-router/src/main/kotlin/azure/ActionHistory.kt

```diff
@@ -416,7 +446,9 @@ class ActionHistory {
             insertReportFile(it, txn)
         }
         reportsOut.values.forEach {
-            insertReportFile(it, txn)
+            if (it.itemCount > 0) {
```

Comment:
What happened to a reportOut value with 0 itemCount before?
Does this change represent a change in functionality - we would potentially have fewer reports tracked.
How could something end up in reportOut.values that we don't actually want to track?

Reply:
let me take another look at this, this change may not be needed anymore.

Before this reportOut would never contain a report with a 0 itemCount, they were filtered out before this. So they essentially vanished